### PR TITLE
Add `include_labels` and `exclude_labels` when exporting data via `dataset.to_*`

### PR DESCRIPTION
**Pull request by dsmilkov**
This enables the user to download a subset of the dataset where some labels are gone.

https://lilac-386213--staging-h3b3igsj.web.app/datasets/dataset_export.html#labels

Towards https://github.com/lilacai/lilac/issues/761

### CODE REVIEW

File: docs/datasets/dataset_export.md [hunk 1]
```diff
@@ -22,47 +22,51 @@ Click "Download" to download the results as a JSON file from the browser.
...
-                              limit=limit,
-                              offset=offset)
-  print(list(items))
-
-  offset += limit
+dataset.to_json('imdb.jsonl', jsonl=True, filters=[('text.lilac/toxicity/gte-small.score', 'less', 0.3)])
 ```
 
-Output (each row is a chunk of results):
+### Labels
```
Comment (nsthorat) [inline docs/datasets/dataset_export.md:59]: maybe put this before filters

File: scripts/deploy_website.py [hunk 2]
```diff
@@ -23,25 +20,15 @@
...
-@click.option(
-  '--firebase_token',
-  help='The firebase token for authentication. Generate this with `firebase login:ci`.',
-  type=str,
-  required=False)
-def main(staging: bool, firebase_token: Optional[str] = None) -> None:
+def main(staging: bool) -> None:
   """Generate a web client from the OpenAPI spec."""
-  firebase_token = firebase_token or env('FIREBASE_TOKEN')
-  if not firebase_token:
```
Comment (nsthorat) [inline scripts/deploy_website.py:34]: i wouldnt revert this -- this makes it much much simpler if your token is expired. the other option is quite annoying (getting token independently and then putting it in ENV)

**Review by nsthorat — APPROVED** [review]
nice work, lgtm

File: scripts/deploy_website.py (hunk 2 above)
Comment (dsmilkov) [reply scripts/deploy_website.py:34]: Service account keys never expire, whereas tokens are 24hr only. Also it says --token is deprecated and will be removed in a future version and will be replaced with service account keys

Comment (nsthorat) [reply scripts/deploy_website.py:34]: Gotchya, can you document how you do use the service account?

File: docs/datasets/dataset_export.md (hunk 1 above)
Comment (dsmilkov) [reply docs/datasets/dataset_export.md:59]: done

File: scripts/deploy_website.py (hunk 2 above)
Comment (dsmilkov) [reply scripts/deploy_website.py:34]: I reverted and always call `firebase auth --reauth` when you deploy -- works all the time, and re-logging is that bad.

**Pull request merged**